specification/cognitiveservices/OpenAI.Inference/models/audio/audio_speech.tsp

```diff
@@ -0,0 +1,77 @@
+import "@typespec/versioning";
+import "@typespec/http";
+
+using TypeSpec.Versioning;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("The available voices for text-to-speech.")
+@added(ServiceApiVersions.v2024_02_15_Preview)
+enum AudioSpeechVoice {
+  @doc("The Alloy voice.")
+  alloy: "alloy",
+
+  @doc("The Echo voice.")
+  echo: "echo",
+
+  @doc("The Fable voice.")
+  fable: "fable",
+
+  @doc("The Onyx voice.")
+  onyx: "onyx",
+
+  @doc("The Nova voice.")
+  nova: "nova",
+
+  @doc("The Shimmer voice.")
+  shimmer: "shimmer",
+}
+
+@doc("The supported audio output formats for text-to-speech.")
+@added(ServiceApiVersions.v2024_02_15_Preview)
+enum AudioSpeechOutputFormat {
+  @doc("Use MP3 as the audio output format. MP3 is the default, general-purpose format.")
+  mp3: "mp3",
+
+  @doc("Use Opus as the audio output format. Opus is optimized for internet streaming and low latency.")
+  opus: "opus",
+
+  @doc("Use AAC as the audio output format. AAC is optimized for digital audio compression and is preferred by YouTube, Android, and iOS.")
+  aac: "aac",
+
+  @doc("Use FLAC as the audio output format. FLAC is a fully lossless format optimized for maximum quality at the expense of size.")
+  flac: "flac",
+}
+
+@doc("A representation of the request options that control the behavior of a text-to-speech operation.")
+@added(ServiceApiVersions.v2024_02_15_Preview)
+model AudioSpeechOptions {
```

Comment:
should we add a `model` option? see the example in https://platform.openai.com/docs/guides/text-to-speech?lang=curl and the definition in transcription in https://github.com/Azure/azure-rest-api-specs/blob/95e98dff110926cee12cea1938f898402229882d/specification/cognitiveservices/OpenAI.Inference/models/audio/audio_transcription.tsp#L84

Reply:
added